pkg/util/executable/executable.go

```diff
@@ -14,21 +14,38 @@ import (
 	"github.com/kardianos/osext"
 )
 
-func path() (string, error) {
+func path(skipSymlinks bool) (string, error) {
 	here, err := osext.Executable()
 	if err != nil {
 		return "", err
 	}
+	retstring, err := filepath.EvalSymlinks(here)
+	if err != nil {
+		if skipSymlinks {
```

Comment:
Since `skipSymlinks` is only done if `EvalSymlinks` fails, this means that `FolderNoSymlinks` will actually resolve symlinks if it can. The way it's used prevents this (since `FolderNoSymlinks` is only called if `Folder` fails), but it's a bit confusing reading the code.